java-checks/src/main/java/org/sonar/java/filters/LombokFilter.java

```diff
@@ -117,11 +124,16 @@ public void visitClass(ClassTree tree) {
     excludeLinesIfTrue(generatesEquals, tree, EqualsNotOverriddenInSubclassCheck.class, EqualsNotOverridenWithCompareToCheck.class);
     excludeLinesIfTrue(generatesPrivateConstructor(tree), tree, UtilityClassWithPublicConstructorCheck.class);
     excludeLinesIfTrue(usesAnnotation(tree, UTILITY_CLASS), tree, BadFieldNameCheck.class, ConstantsShouldBeStaticFinalCheck.class);
-    excludeLinesIfTrue(usesAnnotation(tree, Collections.singletonList("lombok.Value")), tree, FieldModifierCheck.class);
-
```

Comment:
If the rule targets only S1165, we should not change the behavior of other rules.
This change makes sense though, you could keep it and rename the PR to better reflect this, state that you are targetting Lombok's annotations modifying fields modifier for example.

Reply:
PR title updated